docs/source/notes/modules.rst

```diff
@@ -0,0 +1,433 @@
+.. _modules:
+
+Modules
+=======
+
+PyTorch uses modules to both build and represent neural networks. Modules are:
+
+* **Composable building blocks of stateful computation.**
+  PyTorch provides a robust library of modules and makes it easy to define new custom modules, allowing for
+  simple construction of complex, multi-layer neural networks.
```

Comment:
super nit: I'd probably swap where you say easy and simple, "allowing for simple construction" makes it sound like you maybe can't do complicated things.

Reply:
Following the prohibition on the word "complex," mentioned below, may want to strike the word "complex" here, too. It could just be removed or replaced with "modern" or some other exciting word.

Reply:
How about "elaborate"?